CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Requests with empty body and application/json Content-Type Header will now
   return 400 error instead of 500 error.
   [cyberark/conjur#1968](https://github.com/cyberark/conjur/issues/1968)
+- Conjur now properly sets the 'operation' field of a secret fetch audit message.
```

Comment:
can we add the issue here as well?

also - i would remove the seconds sentence.

Reply:
As I have 3 PR which all will have the same comment, I will write in one, merge it to master and then for the rest I will already take the one from master